kfs-core/src/main/resources/edu/arizona/kfs/sys/ojb-sys.xml

```diff
@@ -0,0 +1,41 @@
+<descriptor-repository version="1.0">
+
+	<class-descriptor class="org.kuali.kfs.sys.businessobject.Building" table="SH_BUILDING_T">
+	    <field-descriptor name="campusCode" column="CAMPUS_CD" jdbc-type="VARCHAR" primarykey="true" index="true" />
+	    <field-descriptor name="buildingCode" column="BLDG_CD" jdbc-type="VARCHAR" primarykey="true" index="true" />
+	    <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true" />
+	    <field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true" />
+	    <field-descriptor name="buildingName" column="BLDG_NM" jdbc-type="VARCHAR" />
+		<field-descriptor name="buildingStreetAddress" column="BLDG_STR_ADDR" jdbc-type="VARCHAR"/>
+		<field-descriptor name="buildingAddressCityName" column="BLDG_ADDR_CTY_NM" jdbc-type="VARCHAR"/>
+		<field-descriptor name="buildingAddressStateCode" column="BLDG_ADDR_ST_CD" jdbc-type="VARCHAR"/>
+		<field-descriptor name="buildingAddressZipCode" column="BLDG_ADDR_ZIP_CD" jdbc-type="VARCHAR"/>
+		<field-descriptor name="alternateBuildingCode" column="ALTRNT_BLDG_CD" jdbc-type="VARCHAR"/>
+		<field-descriptor name="buildingAddressCountryCode" column="BLDG_ADDR_CNTRY_CD" jdbc-type="VARCHAR"/>
+		<field-descriptor name="active" column="ROW_ACTV_IND" jdbc-type="VARCHAR" conversion="org.kuali.rice.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion"/>
+	    <reference-descriptor name="extension" class-ref="edu.arizona.kfs.sys.businessobject.BuildingExtension" auto-retrieve="true" auto-update="object" auto-delete="object" proxy="false">
+		    <foreignkey field-ref="campusCode" />
+		    <foreignkey field-ref="buildingCode" />
+	    </reference-descriptor>
+	</class-descriptor>
+	
+	<class-descriptor class="edu.arizona.kfs.sys.businessobject.BuildingExtension" table="SH_BUILDING_EXT_T">
+	    <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true" />
+	    <field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true" />
+	    <field-descriptor name="campusCode" column="CAMPUS_CD" jdbc-type="VARCHAR" primarykey="true" index="true" />
+	    <field-descriptor name="buildingCode" column="BLDG_CD" jdbc-type="VARCHAR" primarykey="true" index="true" />
+	    <field-descriptor name="routeCode" column="ROUTE_CODE" jdbc-type="VARCHAR" />
+	    <reference-descriptor name="routeCodeObj" class-ref="edu.arizona.kfs.sys.businessobject.RouteCode" auto-retrieve="true" auto-update="object" auto-delete="object" proxy="false">
```

Comment:
Should be:
reference-descriptor name="routeCodeObj" class-ref="edu.arizona.kfs.sys.businessobject.RouteCode" auto-retrieve="true" auto-update="none" auto-delete="none" proxy="true"